server/app/controllers/__init__.py

```diff
@@ -1,6 +1,6 @@
 from flask import Blueprint
 
-student_bp = Blueprint('student', __name__)
+student_bp = Blueprint('student', __name__,url_prefix='/api')
 project_bp = Blueprint('project',__name__,url_prefix='/api')
```

Comment:
The url_prefix was deleted in Akram's PR, let's make sure to add it again in the future

Reply:
we should keep the prefix

Reply:
yeah we should it keep, I prefer using the prefix as well